include/spm.rst

```diff
@@ -0,0 +1,44 @@
+.. _lib_spm:
+
+Secure Partition Manager (SPM)
+##############################
+
+The Secure Partition Manager (SPM) uses the SPU peripheral to configure security attributions for the flash, SRAM, and peripherals.
```

Comment:
Could you please use, also, here the ARM spefici terminology?
SPU is the nRF version of IDAU (Implementation-Defined Security Attribution Unit)